### ISSUE TITLE
Add possibility to retrigger a failed deployment

### ISSUE THREAD

**Issue by Gerschtli** [cachix/cachix#491]
Sometimes deployments fail because of network or other temporary issues. It would be great to have a possibility to retrigger deployment in the Web UI and optionally also via cachix CLI.

**Comment by domenkozar** [comment]
Would it be fine if that triggered a new deployment with the same inputs?

**Comment by Gerschtli** [comment]
Yes, and maybe it would be nice to show somewhere (e.g. in a modal box) when you are trying to retrigger a deployment that is not the most recent one. The idea behind this is to ensure that one does not roll back to an old deployment by accident when the intention is just to let the latest deployment succeed.
